automation/01-install-rucio.bash

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env bash
+set -e
+
+# Minikube
+# Possible values for memory: max | format: <number>[<unit>], where unit = b, k, m or g, e.g., 4000mb
+MINIKUBE_MEMORY_RAM=8000mb
+# Possible values for CPUs: max | 1, 2, 3, 4, 5, ...
+MINIKUBE_CPU=4
+
+echo "# --------------------------------------"
+echo "# Check installed packages"
+echo "# --------------------------------------"
+KUBECTL_SUCCESS="The kubectl package is installed."
+KUBECTL_ERROR="The kubectl package is not installed. Please, follow this guide https://kubernetes.io/docs/tasks/tools/install-kubectl/"
+type kubectl &>/dev/null && echo "${KUBECTL_SUCCESS}" || echo "${KUBECTL_ERROR}"
+
+MINIKUBE_SUCCESS="The minikube package is installed."
+MINIKUBE_ERROR="The minikube package is not installed. Please, follow this guide https://minikube.sigs.k8s.io/docs/start/"
+type minikube &>/dev/null && echo "${MINIKUBE_SUCCESS}" || echo "${MINIKUBE_ERROR}"
+
+HELM_SUCCESS="The helm package is installed."
+HELM_ERROR="The helm package is not installed. Please, follow this guide https://helm.sh/docs/intro/install/"
+type helm &>/dev/null && echo "${HELM_SUCCESS}" || echo "${HELM_ERROR}"
+
+echo ""
+echo "# --------------------------------------"
+echo "# Celan local environment"
+echo "# --------------------------------------"
+
+echo "┌──────────────────────────────────────────────┐"
+echo "⟾ Check local Kubernetes clusters for minikube │"
+echo "└──────────────────────────────────────────────┘"
+WILL_STOP_MINIKUBE="n"
+MINIKUBE_RUNNING_KUBERNETES="The minikube is running local Kubernetes clusters."
+minikube status | grep "Running" &>/dev/null && {
+  echo "${MINIKUBE_RUNNING_KUBERNETES}"
+  minikube status
+  read -rp "Do you want to stop the local Kubernetes clusters? (y/N): " WILL_STOP_MINIKUBE
+}
+if [[ "${WILL_STOP_MINIKUBE,,}" == "y" ]]; then
+  minikube stop
+fi
+
+WILL_DELETE_MINIKUBE="n"
+MINIKUBE_HAS_KUBERNETES="The minikube has local Kubernetes clusters."
+minikube status | grep "Stopped" &>/dev/null && {
+  echo "${MINIKUBE_HAS_KUBERNETES}"
+  minikube status || true
+  read -rp "Do you want to delete the local Kubernetes clusters? (y/N): " WILL_DELETE_MINIKUBE
+}
+if [[ "${WILL_DELETE_MINIKUBE,,}" == "y" ]]; then
+  minikube delete --all=true
+fi
+
+MINIKUBE_HAS_NOT_PROFILE="The minikube not has profile. It will create a new one."
```

Comment:
The minikube does not have a profile.

Reply:
Fixed in the commit 84bf9a925933eebb4b73482beb79825180033d7d.